### ISSUE TITLE
Doc Accessibility ACC-005 :  Insufficient colour contrast for the table of content title (Moderate)

### ISSUE THREAD

**Issue by tfrancart** [sparna-git/shacl-play#324]

**Comment by tfrancart** [comment]
Reported on ReSpec side : https://github.com/speced/respec/issues/5339
